app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,88 @@
+.overlay-container {
+  background: #000000;
+  height: 23rem;
+  overflow: hidden;
+  padding-left: 15%;
+  position: relative;
+}
+
+.overlay-container .overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 4rem 2rem 2rem 1rem;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.